addon/loaders/css.js

```diff
@@ -20,7 +20,12 @@ export default nodeLoader(function css(uri) {
     }
 
     // Try using the default onload/onerror handlers...
-    const link = createLoadElement('link', resolve, reject);
+    const link = createLoadElement('link', resolve, function() {
+      if (this.parentNode) {
+        this.parentNode.removeChild(this);
```

Comment:
Is `this` guaranteed here? Does `createLoadElement` call it purposefully bound?

Reply:
I'm not 100% sure :S 
Maybe its better to use the first arg (`event.target`) instead of `this`.

Reply:
It sets the `resolve`/`reject` functions to the `onload`/`onerror` attributes, so it will have the element context. Tests would fail if that doesn't work, so this seems fine to me unless we want to be a bit more specific about what `this` is.